webpack/stories/docs/addingNewComponent.md

```diff
@@ -0,0 +1,98 @@
+# Adding components
+
+## Where to put them
+
+Components are stored in `webpack/assets/javascripts/react_app/components/`. Each component should be placed in its own subfolder that respects the following structure:
+
+```
+─ components/<COMPONENT_NAME>/
```

Comment:
Selectors are missing here

Reply:
I added `<COMPONENT_NAME>Selectors.js` down below. Selectors probably deserve their own paragraph, but I prefer adding it outside of this PR.

Reply:
@tstrachota I am also missing `<COMPONENT_NAME>Consts.js` where constants such as action-types are stored.

Reply:
Good point @ripcurld, added